egs2/chime7_task1/asr1/local/gen_task1_data.py

```diff
@@ -0,0 +1,517 @@
+import argparse
+import glob
+import json
+import os
+from copy import deepcopy
+from datetime import datetime as dt
+from pathlib import Path
+
+import jiwer
```

Comment:
In my understanding, `jiwer` is not automatically installed in the environment. I should manually install it. Could you specify the version?

Reply:
Yes that was my error, I added it now to install_dependecies.sh